src/styles/index.scss

```diff
@@ -1,3 +1,71 @@
+$day-size: 100px;
+$calendar-width: ($day-size + 1px) * 7;
+$days: (
+  'mon': 5,
+  'tue': 101px,
```

Comment:
In the `$days` map, the value for 'mon' is given as `5` without a unit, while 'tue' has `101px`. Ensure all values in the map have consistent units (e.g., `px`) for consistency and to avoid potential issues in calculations.